bin/eks-mount-efs.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Copyright (c) 2016-2017 ForgeRock AS. Use of this source code is subject to the
+# Common Development and Distribution License (CDDL) that can be found in the LICENSE file
+#
+# Sample script to create a Kubernetes cluster on Elastic Kubernetes Service (EKS)
```

Comment:
This script does not create a cluster, but mounts efs to the nodes.

Reply:
well spotted, I copied and pasted that script :)